replay-assets/replay_command_handlers.js

```diff
@@ -2351,18 +2351,19 @@ function convertCdpToRrpCssRules(nodeObj, cdpMatchedStyles) {
     addCdpRule(cdpRule.rule);
   }
 
-  for (const cdpInheritedEntry of inheritedEntries) {
-    // see https://chromedevtools.github.io/devtools-protocol/tot/CSS/#type-InheritedStyleEntry
-    const {
-      // inlineStyle, // inherited inline style
-      matchedCSSRules  // inherited non-inline rules
-    } = cdpInheritedEntry;
+  // [RUN-2999] Our devtools client queries `inheritedEntries` separately.
```

Comment:
Holger confirmed that we should not send `inheritedStyles` for all nodes, since we will query styles of all ancestor elements already, so this would just multiple the amount of data sent.

Reply:
is it being left in to make it easier to revert in the case it causes problems?  I'd vote for removing it :axe: